backend/game_handler.py

```diff
@@ -49,6 +106,15 @@ def reset_game(self):
             self.min_time, np.random.normal(self.average, self.std)
         )
 
+    def bid(self, ws, amount):
+        cash_available = self.game_handler_parent.cash[ws]
+        bid_amount = min(amount, cash_available)
+
+        # Players can only bid once
+        if ws not in self.bids and bid_amount > 0:
+            self.bids[ws] = bid_amount
+            self.game_handler_parent.cash[ws] -= bid_amount
```

Comment:
Same here - I think this is good for now.

In the future, we should allow players to update their bids.